warplib/maths/external_input_check_address.cairo

```diff
@@ -1,12 +1,6 @@
-from starkware.cairo.common.math_cmp import is_le_felt
-from starkware.cairo.common.uint256 import Uint256
 
-func warp_external_input_check_address{range_check_ptr}(x: felt) {
-    let inRange: felt = is_le_felt(
-        x, 0x7ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
-    );
-    with_attr error_message("Error: value out-of-bounds. Value must be less than 2**251.") {
-        assert 1 = inRange;
-    }
-    return ();
+fn warp_external_input_check_address(x: felt) {
+    // The L2 address upper bound is 2**256 - 256, in max is stored its hex representation
+    let max: felt = 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff00;
+    assert( x <= max, 'Error: value out-of-bounds');
```

Comment:
Nit: I would put the error message to include the "address" word somewhere, to make the message more helpful

Also in the comment I would not use generic L2 but StarkNet.

Finally in the comment, upper bound is 2**251 -  256. You had a typo there